### PR TITLE
Print image template

### PR DESCRIPTION
**Pull request by dagewa**
Make it easier to match experiments to imagesets in a multi-imageset
context (such as after cosym/scaling). Related to #1132

### CODE REVIEW

File: command_line/show.py [hunk 1]
```diff
@@ -258,6 +258,9 @@ def show_experiments(experiments, show_scan_varying=False):
             text.append("Format class: %s" % format_class.__name__)
         if expt.identifier != "":
             text.append("Experiment identifier: %s" % expt.identifier)
+        template = expt.imageset.get_template()
```
Comment (rjgildea) [inline command_line/show.py]: I may be wrong, but ISTR that `get_template()` only exisits for `ImageSequence`, not `ImageSet`?

Comment (dagewa) [reply command_line/show.py]: Yes, it seems you are right

Comment (graeme-winter) [reply command_line/show.py]: Even so the addition would be useful, could `try` / `except` or `hasattr` this one?

Comment (dagewa) [reply command_line/show.py]: Yep, done in 5ba4f498d8 :+1:

**Pull request merged**